docs/site/src/content/docs/content/assets-views.mdx

```diff
@@ -259,6 +259,73 @@ Within a template, the `_index_` string can be used to substitute the array-inde
 }
 ```
 
+### Template ordering
+
+When writing template content, the position of the template in relation to non-template content is not guaranteed when it gets transformed into its final JSON output. Because of this, templates support a `placement` property that specifies whether the template should expand before or after pre-existing content.
```

Comment:
Technically the DSL does preserve the ordering, but JSON itself  doesn't guarantee preserving the key ordering.